understanding/20/sensory-characteristics.html

```diff
@@ -11,12 +11,12 @@ <h1>Understanding Sensory Characteristics</h1>
    <section id="brief">
 		<h2>In brief</h2>
 		<dl>
-		<dt>Objective</dt><dd>Instructions are understandable by more people</dd>
-		<dt>Author task</dt><dd> Describe controls by name, not just by appearance or location</dd>
-		
-		</dl>
-		
-		</section>
+         <dt>Goal</dt><dd>Instructions are understandable by more people.</dd>
+         <dt>What to do</dt><dd> Describe controls by name, not just by appearance or location.</dd>
+         <dt>Why it's important</dt><dd>People who are blind or have low vision need non-visual instructions.</dd> 
```

Comment:
@rainbreaw commented
> Consider including people with visual processing disorders.

Reply:
@GreggVan stated
> Agree
That would make it
`<dt>Why it's important</dt><dd>People who are blind or have low vision or other visual processing disabilities need non-visual instructions.</dd>`

Reply:
@rainbreaw, I agree this is an easy addition; however, we again encounter the problem that nothing in the Understanding document, including in the Benefits section, mentions "visual processing disabilities".

The In Brief language is not intended to introduce new concepts into an Understanding document. So I recommend generating a PR for the Understanding document that incorporates this information into the main document content _and_ adds in a change to the Why statement.